lte/gateway/c/core/oai/test/spgw_task/BUILD.bazel

```diff
@@ -72,3 +72,12 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_test(
+    name = "spgw_service_impl_test",
```

Comment:
Can we add `size = "small,"` to this test? This ensures that a timeout of 60 seconds is applied (assuming 60 seconds is sufficient, if not a larger test size can be used).  This way timeouts in CI happen without enormous blocking times.